### PR TITLE
Build Early Access wallpaper

### PR DESCRIPTION
**Pull request by danirabbit**
Adds the early access wall with a build option.

Builds with https://github.com/elementary/wallpapers/tree/deb-packaging-daily

Closes https://github.com/elementary/default-settings/issues/80
Closes https://github.com/elementary/prerelease-settings/issues/6

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,25 +1,35 @@
...
+
     - name: Build
       run: |
         meson build
-        ninja -C build
+        ninja -C build install
+
+    - name: Build (Early Access)
+      run: |
+        meson configure -Dearly_access=true
```
Comment (meisenzahl) [inline .github/workflows/main.yml:34]:
```suggestion
        meson configure build -Dearly_access=true
```

This does not work as expected. From meson's docs:

> You invoke `meson configure` by giving it the location of your build dir. If omitted, the current working directory is used instead.

Source: https://mesonbuild.com/Configuring-a-build-directory.html

**Review by meisenzahl — CHANGES_REQUESTED** [review]
Just one small suggestion to make CI run as intended. Otherwise, it works as expected. Can you modify the build parameters via Launchpad or how do you want to use this feature for daily builds?

File: .github/workflows/main.yml (hunk 1 above)
Comment (danirabbit) [reply .github/workflows/main.yml]: Oops thanks for catching that

**Comment by danirabbit** [comment]
@meisenzahl Yeah so I have a packaging branch which sets this build option so the daily recipe will use the daily packaging and the stable recipe will use the stable packaging :)

**Review by meisenzahl — APPROVED** [review]
LGTM

**Pull request merged**